src/lib/components/molecules/party-profile/style.module.css

```diff
@@ -0,0 +1,74 @@
+.text {
+  font-family: var(--text-sans);
+  font-size: var(--sans-small);
+  line-height: var(--sans-line-height);
+  font-weight: 700;
+  margin-left: var(--space-1);
+}
+
+.container {
+  font-weight: 700;
+  min-width: 290px;
+  max-width: 530px;
+  margin-right: 6px;
+}
+
+.title {
+  font-family: var(--text-headline);
+  font-size: var(--headline-xsmall);
+  color: var(--primary-text-color);
+  font-weight: 500;
+  line-height: 1.1;
+
+  &:before {
+    content: '';
+    height: 4px;
+    width: 2.2em;
+    border-radius: 50px;
+    display: block;
+  }
+
+}
+
+.subtitle {
+  font-family: var(--text-headline);
+  font-size: var(--headline-xsmall);
+  color: var(--primary-text-color);
+  font-style: italic;
+  font-weight: 500;
+  line-height: 1.3;
+
+}
+
+.blurb {
+  font-family: var(--text-serif);
+  font-size: var(--body-small);
+  font-weight: 300;
+  margin-right: 6px;
```

Comment:
Could do with a bit of padding or margin top here to space it out more from the title
<img width="346" alt="Screenshot 2024-03-20 at 14 31 30" src="https://github.com/guardian/interactive-component-library/assets/10324129/f7572f45-c42e-4c15-9446-0feec13c1700">